templates/taxi/car_detail.html

```diff
@@ -3,20 +3,26 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h1>
+    Drivers
+    {% if user not in car.drivers.all %}
+      <a style="float: right" href="{% url 'taxi:assign-delete-drivers' pk=car.id %}" class="btn btn-success">Assign me to this car</a>
+    {% else %}
+      <a style="float: right" href="{% url 'taxi:assign-delete-drivers' pk=car.id %}" class="btn btn-danger">Delete me from this car</a>
+    {% endif %} 
```

Comment:
Better not to use buttons inside header tags

Reply:
Thanks, changed.